src/test/java/seedu/address/testutil/TypicalEvents.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.testutil;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.event.Event;
+
+/**
+ * A utility class containing a list of {@code Events} objects to be used in tests.
```

Comment:
```suggestion
 * An utility class containing a list of {@code Events} objects to be used in tests.
```

Reply:
https://www.reddit.com/r/ENGLISH/comments/3rnp84/why_is_it_aeh_utility_not_an_utility/ Seems like it shouldn't be 'An' because of the pronunciation